Index: SConstruct
===================================================================
--- SConstruct	(版本 6274)
+++ SConstruct	(工作副本)
@@ -129,7 +129,7 @@
   'gcc': {
     'all': {
       'CCFLAGS':      ['$DIALECTFLAGS', '$WARNINGFLAGS'],
-      'CXXFLAGS':     ['$CCFLAGS', '-fno-rtti', '-fno-exceptions'],
+      'CXXFLAGS':     ['$CCFLAGS'],
     },
     'visibility:hidden': {
       # Use visibility=default to disable this.
@@ -458,7 +458,7 @@
   'gcc': {
     'all': {
       'LIBPATH': ['.'],
-      'CCFLAGS': ['-fno-rtti', '-fno-exceptions']
+      'CCFLAGS': []
     },
     'os:linux': {
       'LIBS':         ['pthread'],
